src/setup/cortx_deploy

```diff
@@ -94,13 +100,44 @@ validate_config $CLUSTER_INFO
 
 export PATH=$PATH:/opt/seagate/provisioner/bin
 
-# Apply Cluster Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE'"
-cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE";
+# Check system is pre-configured:
+# config apply: check confstore file, if file exsist skip config apply.
+# cluster bootstrap: read overall_mp_status from state file, if success skip cluster bootstrap.
 
-# Apply Common Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c '$CORTX_CONFSTORE'"
-cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c "$CORTX_CONFSTORE";
+if [ -z $SKIP_CHECK ]; then
+    FILE=$(awk -F:// '{print $2}' <<< $CORTX_CONFSTORE)
+    if [ -f "$FILE" ]; then
+        log $INFO "$FILE already exists."
```

Comment:
Message needs to be more meaningful like 
"File $FILE exists. System looks to be pre-configured."

Reply:
Done.